HOWTO-de.md

```diff
@@ -0,0 +1,17 @@
+Lese das hier auch in anderen Sprachen: [English](HOWTO.md), [Français](HOWTO-fr.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md) [Español](HOWTO-es.md), [فارسی](HOWTO-fa_IR.md)
```

Comment:
Please, mantain the same order of items like you can see in other HOWTOs. It contributes to have a good UX.

```markdown
[Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md).
```

Reply:
Changed it! :)